Sources/HipMobileUI.Droid/HipMobileUI.Droid.csproj

```diff
@@ -322,8 +322,31 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <AndroidAsset Include="Assets\fonts\AlteSchwabacher.ttf" />
     <None Include="app.config" />
+    <AndroidAsset Include="Assets\SourceSansPro-Black.ttf" />
```

Comment:
This seems like a really memory inefficient way of including the assets due to the obvious redundancy. For me it's still working if I include the files only in the parent folder (Assets). If you have no objections, I would recommend you to remove the files from the fonts-folder and keep them only in the parent folder (maybe even delete the entire fonts folder). Since we aren't using any other assets so far, this should be fine.

Reply:
I added them under fonts folder first ,them moved to Asset folder,so ...forgot to delete in font folder.... 😄